### PR TITLE
Allows sending message in project to multiple groups of users. Hides other groups in the lowest level.

### PR DESCRIPTION
**Pull request by jerdzica**

### CODE REVIEW

File: addons/bestja_project/__openerp__.py [hunk 1]
```diff
@@ -24,6 +24,7 @@
         'messages.xml',
         'menu.xml',
         'security/ir.model.access.csv',
+        'data.xml',
```
Comment (KrzysiekJ) [inline addons/bestja_project/__openerp__.py]: There is no such file…

File: addons/bestja_project_hierarchy/__openerp__.py [hunk 2]
```diff
@@ -25,5 +25,6 @@
         'messages.xml',
         'security/ir.model.access.csv',
         'security/security.xml',
+        'data.xml',
```
Comment (KrzysiekJ) [inline addons/bestja_project_hierarchy/__openerp__.py]: There is no such file…

File: addons/bestja_project/__openerp__.py (hunk 1 above)
Comment (jerdzica) [inline addons/bestja_project/__openerp__.py]: File added. Sorry:)

**Comment by KrzysiekJ** [comment]
Merged in 497ad869b145ae7d3b97f4ec6903fd2f042096ae and 35987f0ff874603ba3298322ff330fcf24e0af77.

**Pull request closed without merging**